myhpi/static/js/myHPI.js

```diff
@@ -38,75 +38,80 @@ const toggleHideOnScrollBlock = (element) => {
  *   Before, the elements will always be displayed.
  */
 const toggleElementVisibilityOnScroll = (minScrollPosition = 0) => {
-    let currentScrollPosition = window.scrollY
-    let elements = document.querySelectorAll(".xl-hide-on-scroll")
-    if (
-        previousScrollPosition < currentScrollPosition &&
-        currentScrollPosition > minScrollPosition
-    ) {
-        elements.forEach((el) => {
-            if (el.querySelector(".block-ancestor-hide")) return
-            el.classList.add("hide-now")
-        })
-    } else {
-        elements.forEach((el) => el.classList.remove("hide-now"))
-    }
-    previousScrollPosition = currentScrollPosition
+  let currentScrollPosition = window.scrollY
+  let elements = document.querySelectorAll(".xl-hide-on-scroll")
+  if (
+    previousScrollPosition < currentScrollPosition &&
+    currentScrollPosition > minScrollPosition
+  ) {
+    elements.forEach((el) => {
+      if (el.querySelector(".block-ancestor-hide")) return
+      el.classList.add("hide-now")
+    })
+  } else {
+    elements.forEach((el) => el.classList.remove("hide-now"))
+  }
+  previousScrollPosition = currentScrollPosition
 }
 
 const localizeLastPublished = () => {
-    const lastPublished = document.getElementById("last-published")
-    const timezone_server = lastPublished.getAttribute("title")
-    const timezone_user = Intl.DateTimeFormat().resolvedOptions().timeZone
-
-    if(timezone_server !== timezone_user) {
-        lastPublishedLocalized = new Date(lastPublished.getAttribute("datetime")).toLocaleString(undefined, {
-            year: "numeric", month: "numeric", day: "numeric",
-            hour: "numeric", minute: "2-digit",
-            timeZoneName: "short"
-        })
-
-        // replace <time> width <abbr> tag
-        let abbr = document.createElement("abbr");
-        abbr.setAttribute("title", lastPublishedLocalized)
-        lastPublished.removeAttribute("title")
-        lastPublished.innerText = lastPublished.textContent.trim()
-
-        let parent = lastPublished.parentNode;
-        parent.replaceChild(abbr, lastPublished);
-        abbr.appendChild(lastPublished);
-    }
+  const lastPublished = document.getElementById("last-published")
+  const timezone_server = lastPublished.getAttribute("title")
+  const timezone_user = Intl.DateTimeFormat().resolvedOptions().timeZone
+
+  if (timezone_server !== timezone_user) {
+    lastPublishedLocalized = new Date(
+      lastPublished.getAttribute("datetime")
+    ).toLocaleString(undefined, {
+      year: "numeric",
+      month: "numeric",
+      day: "numeric",
+      hour: "numeric",
+      minute: "2-digit",
+      timeZoneName: "short",
+    })
+
+    // replace <time> width <abbr> tag
+    let abbr = document.createElement("abbr")
+    abbr.setAttribute("title", lastPublishedLocalized)
+    lastPublished.removeAttribute("title")
```

Comment:
What does that do?

Reply:
Yeah so I didn't do anything in this file, except for my formatter to change the format

Reply:
This line seems to be missing:` lastPublished.innerText = lastPublished.textContent.trim()`

Reply:
Thank you all for your help to fix this, done :)